spec/src/main/asciidoc/chapters/language/statements.adoc

```diff
@@ -19,22 +21,23 @@ The `from` clause, with syntax given by `from_clause`, specifies an _entity name
 
 NOTE: The syntax of the `update` statement is irregular, with the `from` keyword implied. That is, the syntax _should_ be `update from Entity`, but for historical reasons it is simply `update Entity`.
 
-The `from` clause is optional in `select` statements. When it is missing, the queried entity is determined by the return type of the repository method, or, if the return type is not an entity type, by the primary entity type of the repository.
+The `from` clause is optional in `select` statements. When it is missing, the queried entity is determined by some defaulting rule defined externally to this specification, for example:
 
-For example, this repository method declaration:
+1. a result type might be supplied via an API cainvocationll (for example, via `createQuery()` in Jakarta Persistence), or
+2. it might be determined by static analysis (for example, in Jakarta Data, by the return type of a repository method or by the primary entity type of the repository).
 
-[source,java]
+For example, this query:
```

Comment:
Having removed the parts of the example that show where `Book` came from, we need to explain it,
```suggestion
For example, this query provided to the Jakarta Persistence `createQuery()` method with result type of `Book.class` or supplied to a Jakarta Data repository method that returns `List<Book>`:
```
The alternative would be to leave the Jakarta Data example and state that it is an example of Jakarta Data.